docker/Dockerfile.nvidia

```diff
@@ -124,62 +106,19 @@ RUN if [ "x$MPI_VER" = "x4" ]; then \
         rm -f  $HPCSDK_HOME/comm_libs/mpi && \
         ln -sf $HPCSDK_HOME/comm_libs/hpcx/latest/ompi \
                $HPCSDK_HOME/comm_libs/mpi ; \
-    fi;  \
-    /venv/bin/pip install --no-cache-dir -r /app/requirements-mpi.txt && \
-    rm -rf ~/.cache/pip
+    fi;
 ENV CFLAGS=
 
-ADD ./devito /app/devito
```

Comment:
I'm confused, where are these going?

Reply:
Instead of copying from the host to the image, the Dockerfile clones the latest Devito.
https://github.com/devitocodes/devito/pull/1797/files#diff-712980f1b6ff2592c72da4a45d841c8408275e3b9178043c7db3736d7c30ba68R113